internal/pkg/propagation/propagation.go

```diff
@@ -240,34 +247,28 @@ func (prop *Propagation) taintOperands(n ssa.Node, maxInstrReached map[*ssa.Basi
 	}
 }
 
-func (prop *Propagation) taintCall(call *ssa.Call, maxInstrReached map[*ssa.BasicBlock]int, lastBlockVisited *ssa.BasicBlock) {
+func (prop *Propagation) taintCall(call ssa.Node, callCommon ssa.CallCommon, maxInstrReached map[*ssa.BasicBlock]int, lastBlockVisited *ssa.BasicBlock) {
```

Comment:
(I don't have a hard preference here, just sharing my perception.) AFAIK we've been following the types of whatever `ssa` makes available to us? So e.g. all concrete nodes are pointers, but this CallCommon thing is a value, so I used it as is instead of passing a reference.

WDYT?

Reply:
[outdated, resolving]